src/org/zaproxy/zap/extension/ascanrulesBeta/InsecureHTTPMethod.java

```diff
@@ -260,19 +294,9 @@ public void scan() {
 							AlertThreshold threshold = getAlertThreshold();
 							if (threshold != AlertThreshold.LOW ) {
 								//!= Low threshold --> verify it
-								if (enabledmethod.equals ("TRACE") || enabledmethod.equals ("TRACK")) {
-									if (log.isDebugEnabled ())  log.debug("Verifying a TRACE/TRACK");								
-									MessageAndEvidence mae = testTraceOrTrack(this.getBaseMsg(), enabledmethod);
-									if ( mae != null) {
-										evidence = mae.evidence;
-										alertMessage = mae.message;
-										raiseAlert = true;
-										description = Constant.messages.getString("ascanbeta.insecurehttpmethod.trace.exploitable.desc", enabledmethod);
-										extraInfo = Constant.messages.getString("ascanbeta.insecurehttpmethod.trace.exploitable.extrainfo", evidence);
-									}								
-								} else if (enabledmethod.equals ("CONNECT")) {
+								if (enabledmethod.equals ("CONNECT")) {
 									
-									if (log.isDebugEnabled ())  log.debug("Verifying a CONNECT");
+									log.debug("Verifying a CONNECT");
```

Comment:
The `log.isDebugEnabled()` check should still be performed and should wrap all `log.debug` calls throughout the code.